client/src/components/ListGroup/ListGroup.scss

```diff
@@ -44,7 +44,7 @@
 }
 
 .list-group-item-text {
-  line-height: $line-height;
+  line-height: $line-height-base;
```

Comment:
inconsistent, other references to `$line-height-base` adds a `* 1rem` to the end